data/resistance intro.txt

```diff
@@ -0,0 +1,1187 @@
+			`	You say your goodbyes, and you head back to your ship. Time to head back to Freya.`
+				accept
+	on complete
+		conversation
+			`Freya messages you as you're coming for a landing, saying she will be waiting in the spaceport to meet up.`
+
+
+
+mission "House Bebliss 5"
+	name "An Answer to House Bebliss"
+	description "Now that Freya has agreed to share her insights with the members of House Bebliss, head to <destination>, to figure out if and how they will meet."
+	to offer
+		has "House Bebliss 4: done"
+	source "Pugglemug"
+	destination "Factory of Eblumab"
+	on offer
+		conversation
+			`Freya quickly approaches you as you step foot on the spaceport, and has you follow her around the city, until you come back to where you first found her and her team working on Pug technology.`
+			`	You see the Arach suitcase is on top of a table, empty, and several schematics, blueprints and research papers, scattered around it.`
+			`	"I assume they translated these, the words are all out of conventional order," she says, shuffling around some of the papers. "Why are they even asking for me? These aliens know way more than I imagined, from what you were saying they wanted from me..." She sighs, still looking at the papers.`
+			choice
+				`	"Are you alright?"`
+					goto alright
+				`	"Have you decided to help them?"`
+					goto helpspider
+			label alright
+			`	"I'm fine, it's just... These all made me feel kind of impotent as I went through them, is all."`
+				goto helpspider
+			label helpspider
+			`	She looks back at you, then picks up some papers and books of her own from a drawer.`
+			`	"Bring these to those spider aliens, they're copies of everything I've found out here. If they want more, well, I'm not migrating to live among stupidly advanced aliens. Not yet, anyway. So they'd need to come here and research for themselves."`
+			`	She thanks you for bringing her the information, and you two say goodbye as you head back to your ship.`
+				accept
+
+
+
+mission "House Bebliss 6"
+	name "Jumping Spiders"
+	description "Acquire a spare Jump Drive, and bring it to <destination>, where the House Bebliss representatives will have it secretly installed in one of their ships."
+	landing
+	to offer
+		has "House Bebliss 5: done"
+	source "Factory of Eblumab"
+	destination "Blubipad's Workshop"
+	on offer
+		conversation
+			`You once again land close to the Arachi estate, as you prepare to bring them Freya's findings and her response.`
+			`	"To work with us, acepted, has she, Captain?"`
+			`	You tell them about Freya's response to their own findings, and hand them the copies of her studies, as you explain she has decided to not leave Deneb.`
+			`	"Understandable, that is," one of them says, as they all look over her findings. Although it's all written in a language they don't understand, they don't seem to have a problem staring for several minutes at the schematics she drew there.`
+			`	Finally, they
```

Comment:
This event actually took place 2 in-game years after I finished the previous mission. My problem with this is that it is a massive amount of time to wait in between missions. I would recommend a 1-2 month break maximum so as to keep the player interested.

Reply:
I think it could be longer than 1-2 months, but 2 years is long enough where it's easy to forget what was happening. Maybe 5-6 months?

Reply:
cut it down from 690 to 140